src/test/java/seedu/address/model/explorer/FileListStub.java

```diff
@@ -0,0 +1,5 @@
+package seedu.address.model.explorer;
+
+public class FileListStub extends FileList {
```

Comment:
What is this FileListStub for? I can't find any class that use this. If this class is not being used, can it be deleted? We can add it again later if we really need it, to make the code cleaner.

Reply:
Oh, yeah, forgot to delete that